core/tests/puppeteer-acceptance-tests/specs/curriculum-admin/create-publish-unpublish-and-delete-topic-and-skill.spec.ts

```diff
@@ -19,43 +19,31 @@
 import {UserFactory} from '../../utilities/common/user-factory';
 import testConstants from '../../utilities/common/test-constants';
 import {CurriculumAdmin} from '../../utilities/user/curriculum-admin';
-import {ExplorationEditor} from '../../utilities/user/exploration-editor';
+import {LoggedInUser} from '../../utilities/user/logged-in-user';
 
 const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
 const ROLES = testConstants.Roles;
 
 describe('Curriculum Admin', function () {
-  let curriculumAdmin: CurriculumAdmin & ExplorationEditor;
-  let explorationId: string | null;
+  let curriculumAdmin: CurriculumAdmin;
+  let guestUser: LoggedInUser;
 
   beforeAll(async function () {
     curriculumAdmin = await UserFactory.createNewUser(
       'curriculumAdm',
       'curriculum_admin@example.com',
       [ROLES.CURRICULUM_ADMIN]
     );
+
+    guestUser = await UserFactory.createNewUser(
+      'guestUser1',
+      'guest_user1@example.com'
+    );
   }, DEFAULT_SPEC_TIMEOUT_MSECS);
 
   it(
     'should create and publish topics, subtopics, skills, stories and chapters.',
```

Comment:
I don't think it includes creating stories and chapters? Please rewrite this description to explain the test.

Reply:
Done.